src/Serilog.Sinks.MongoDB/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The MongoDB sink for Serilog",
+  "authors": [ "Kiran Makkapati", "Serilog Contributors" ],
+  "tags": [ "serilog", "mongodb" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": "2.0.0-beta-511",
+    "Serilog.Sinks.PeriodicBatching": "2.0.0-beta-700"
+  },
+
+  "frameworks": {
+    "dnx46": {
```

Comment:
Thanks, yes, was thinking this is one we sort out at RC2 (though the list of RC2 tech-debt is getting long...) :-)